spec/features/regions_spec.rb

```diff
@@ -17,10 +17,27 @@
   scenario 'try to add a duplicate region' do
     region = build(:region, name: 'Antioquia')
 
-    expect( create_region(region) )
-    expect( create_region(region) )
+    2.times do
+      expect( create_region(region) )
+    end
+
     expect(page).to have_content 'Name has already been taken'
+  end
 
+  scenario 'edits a region' do
+    create(:region)
+    update_region
```

Comment:
I don't see any particular reason to extract `update_region` and `destroy_region` into a helper macro.

In general we only need to extract when we start seeing duplication, for now I think is better to have those feature specs explicitly defined on each rspec scenario